src/practices/LanguageIndependent/DoesPullRequests.spec.ts

```diff
@@ -33,6 +33,7 @@ describe('DoesPullRequests', () => {
 
     const evaluated = await practice.evaluate(containerCtx.practiceContext);
     expect(evaluated).toEqual(PracticeEvaluationResult.practicing);
+    expect(practice.data.statistics?.pullRequests).not.toBeUndefined();
```

Comment:
Why not to use: `expect(practice.data.statistics?.pullRequests).toBeDefined()` ?